src/licenses/LicenseModel.cpp

```diff
@@ -0,0 +1,184 @@
+#include "LicenseModel.h"
+
+#include <QDirIterator>
+
+static const char *const platformRootPath = ":/licenses";
```

Comment:
the static cost can also be converted to `constexpr`

Reply:
I moved them into an anonymous namespace instead of having them static